standalone/src/models/cart/cart/cart/cart-draft/fields-config.ts

```diff
@@ -46,6 +46,7 @@ const commonFieldsConfig = {
 export const restFieldsConfig: TModelFieldsConfig<TCartDraftRest> = {
   fields: {
     ...commonFieldsConfig,
+    priceRoundingMode: oneOf(...Object.values(roundingMode)),
```

Comment:
I see `priceRoundingMode` existing also in the GraphQL schema definition.
Why are we only adding it in the REST mode?